.npmrc

```diff
@@ -0,0 +1,24 @@
+registry = https://registry.npmjs.org
+ignore-scripts = true
+prefer-workspace-packages = true
+
+####################
+# super strict mode
+####################
+auto-install-peers=false
+strict-peer-dependents=true
```

Comment:
@NullVoxPopuli I believe you've [mistyped](https://pnpm.io/npmrc#strict-peer-dependencies) this.

Reply:
appears so!